core/src/components/cat-notification/cat-notification.tsx

```diff
@@ -123,15 +133,55 @@ class CatNotificationService {
       className: 'cat-toastify',
       gravity: position.gravity,
       position: position.position,
-      stopOnFocus: true
+      stopOnFocus: true,
+      offset: {
+        x: '1.5rem',
+        y: '1.5rem'
+      }
+    };
+    Toastify(toastOptions).showToast();
+  }
+
+  primary(title: string, message = ' ', options?: Partial<ToastOptions>): void {
+    const position: ToastPosition = this.getPosition(options);
+    const toastOptions: Options = {
```

Comment:
maybe you could put this option object in private method? to pass the principle of DRY, since it is the same in the other methods

Reply:
done